### PR TITLE
Email updates 2024

### PR DESCRIPTION
**Pull request by dmcavoy**
Update a bunch of small things in workshop emails ahead of summer workshops.

### CODE REVIEW

File: dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %h3
-  Pre-Course Survey
+  Pre-Workshopg Survey
```
Comment (hannahbergam) [inline dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml]: Small typo^

**Review by hannahbergam — COMMENTED** [review]
One small typo, but otherwise seems good!

File: dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml (hunk 1 above)
Comment (dmcavoy) [reply dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml]: grrr... that g which shows up when I start typing git without leaving my editor. haha

Comment (hannahbergam) [reply dashboard/app/views/pd/workshop_mailer/_pre_course_survey.html.haml]: So relatable😅

**Review by hannahbergam — APPROVED** [review]
🙌🏼

**Pull request merged**